### PR TITLE
correct version of stacktrace-js

### PR DESCRIPTION
**Pull request by luisvt**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ sudo: false
 
 language: node_js
 node_js:
-  - '0.10'
+  - '7.0'
```
Comment (defunctzombie) [inline .travis.yml]: not relevant to this PR

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "shallow-copy": "0.0.1",
     "shell-quote": "1.4.1",
     "stack-mapper": "0.2.2",
-    "stacktrace-js": "http://github.com/defunctzombie/stacktrace.js/tarball/07e7b9516f1449f5c209e4f67f11a43f738c1712",
+    "stacktrace-js": "^1.3.1",
```
Comment (defunctzombie) [inline package.json]: notice how this doesn't match any of the other dependencies by using `^`

**Comment by defunctzombie** [comment]
What do you mean by "correct" version?

**Comment by luisvt** [comment]
It looks like that repository was not working yesterday. I think it would be better to have a `semver` version instead having to point to a github repository.

**Comment by defunctzombie** [comment]
This PR cannot be merged until the issues outlined in review are addressed.

File: package.json (hunk 2 above)
Comment (luisvt) [inline package.json]: what do you mean?

**Pull request closed without merging**